Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -118,6 +121,133 @@ def get_current_utc_time() -> datetime:
     return datetime.utcnow()
 
 
+def return_user_filter(user_to_search: str, users_list):
+    """
+    Looks through the inputted list and if the user to search for exists, will return the user. Otherwise, it will
+    return an empty dict
+
+    Args:
+        user_to_search: The user we are searching for
+        users_list: A list of user dictionaries to search through
+
+    Returns:
+        The dict which matched the user to search for if one was found, else an empty dict.
+    """
+    users_filter = list(filter(lambda u: u.get('name', '').lower() == user_to_search
+                               or u.get('profile', {}).get('display_name', '').lower() == user_to_search
+                               or u.get('profile', {}).get('email', '').lower() == user_to_search
+                               or u.get('profile', {}).get('real_name', '').lower() == user_to_search, users_list))
+    if users_filter:
+        return users_filter[0]
+    else:
+        return {}
+
+
+def get_user_by_email(user_to_search: str) -> dict:
+    """
+    Searches for a user when given the user's email.
+
+    Args:
+        user_to_search: the user's email we are searching for.
+    Returns:
+        Formatted user results if a user is found.
+    """
+    if not re.match(emailRegex, user_to_search):
+        raise ValueError('must provide user email')
+
+    _body = {
+        'email': user_to_search
+    }
+    response = send_slack_request_sync(CLIENT, 'users.lookupByEmail', http_verb='GET', body=_body)
+    user = response.get('user', {})
+
+    if not user:
+        err_str = f'User {user_to_search} not found in Slack'
+        if SAFE_MODE:
+            err_str += ' and Safe Mode is enabled. If this command worked previously for you, please try disabling' \
+                       ' Safe Mode from the instance configuration or use the users email instead.'
```

Comment:
please direct to the limitations and warning about disabling safe mode in the readme.